CHANGELOG.md

```diff
@@ -1,80 +1,5 @@
 # Semantic Release Automated Changelog
 
-# [3.1.0](https://github.com/AlaskaAirlines/auro-sidenav/compare/v3.0.1...v3.1.0) (2024-12-22)
```

Comment:
Need to re-instate the changelog comments

Reply:
All the comments are there. If it looks like they are removed. It's because they moved around a little.